.github/workflows/publish.yml

```diff
@@ -0,0 +1,37 @@
+name: Publish Docker image
+
+on:
+  release:
+    types: [published]
+  push:
+    branches: [master, devel]
+
+jobs:
+  push_to_registry:
+    name: Push swift upload runner Docker image to Docker Hub
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Login to DockerHub Registry
+        run: echo '${{ secrets.DOCKER_PASSWORD }}' | docker login -u '${{ secrets.DOCKER_USERNAME }}' --password-stdin
+      - name: Get the version
+        id: vars
+        run: echo ::set-output name=tag::$(echo ${GITHUB_REF:10})
+      - name: Build the tagged Docker image
+        if:  ${{ steps.vars.outputs.tag != '/master' }}
```

Comment:
Should this line be swapped with line 27?
`if:  ${{ steps.vars.outputs.tag != '/master' && steps.vars.outputs.tag != '/devel' }}`

Reply:
done.